pages/blog/google-season-of-docs-2022.md

```diff
@@ -0,0 +1,95 @@
+
+
+### ⌛ Timeline
+The project itself will take approximately 3-6 months to complete, depending on how many technical writers (TW) get involved. If we can get even just 2-4 TWs, then we can work on both migrating the CLI and Tools documentation. But if we can only get 1-2 folks, it would be hard to do more than the migration of CLI or Tools docs. Ideally, the timeline could look like the following:
+
+- **May:** Orientation on how to contribute to AsyncAPI Inititiave, how Docs issues are organized, detail how we're migrating our CLI and Tools Docs, and assign good `first-time-tickets` to get each new TW contributor started. 
+- **June - August:**	Each TW goes through designated issues marked for both first time contributors and work set aside for `GSoD 2022`. Each TW starts creating documentation for their individual issues assigned/selected. 
+- **September - October:** We determine if we're going to be able to complete both CLI and Tools Docs, depending on how many TWs are in our group and how much they've been able to complete so far. We re-align priorities as needed and asses what is missing to reach our 2022 IA change goals for AsyncAPI Docs.
+- **November:**	Project completion and all contributors receive some swag! 
+
+
+### 💸 Project budget
+We have set aside 2 mentors for now, for our 1 project on improving our IA. For this we would like to request a US $5000 budget to Google, should we be selected. 
+
+___ 
+
+## 👉🏽 Get started contributing to AsyncAPI Docs Today
+Last but not least, don't forget that code isn't the only way to contribute to OSS; Dev Docs are a **huge** help that benefit the entire OSS ecosystem. At AsyncAPI, we value Doc contributions as much as every other type of contribution. ❤️
+
+To get started as a Docs contributor:
+1. Familiarize yourself with our [project's Contribution Guide](https://github.com/asyncapi/community/blob/master/CONTRIBUTING.md) and our [Code of Conduct](https://github.com/asyncapi/.github/blob/master/CODE_OF_CONDUCT.md).
+2. Head over to our Docs GH Board [here](https://github.com/orgs/asyncapi/projects/8).
+3. Pick an issue you would like to contribute to and leave a comment introducing yourself. This is also the perfect place to leave any questions you may have on how to get started. 
+4. If there is no work done in that Docs issue yet, feel free to open a PR and get started!
+
+### 🏷 Tag me in your AsyncAPI Doc PRs
+Do you have a documentation contributor question and you're wondering how to tag me into a GitHub discussion or PR? Never fear!
+
+Tag me in your AsyncAPI Doc PRs or [GitHub Discussions](https://github.com/asyncapi/community/discussions/categories/docs) via my GitHub handle, [`/alequetzalli`](https://github.com/alequetzalli) 🐙.
```

Comment:
Preview of the build fails https://app.netlify.com/sites/asyncapi-website/deploys/623a4308d91fae0009430500

```
10:48:12 PM: Failed to compile.
10:48:12 PM: 
10:48:12 PM: ./pages/blog/google-season-of-docs-2022.md:59:16
10:48:12 PM: Syntax error: Unterminated JSX contents.
10:48:12 PM:   57 | <p>{`Tag me in your AsyncAPI Doc PRs or `}<a parentName="p" {...{"href":"https://github.com/asyncapi/community/discussions/categories/docs"}}>{`GitHub Discussions`}</a>{` via my GitHub handle, `}<a parentName="p" {...{"href":"https://github.com/alequetzalli"}}><inlineCode parentName="a">{`/alequetzalli`}</inlineCode></a><undefined parentName="p">{` `}<span {...{"role":"img","aria-label":"octopus"}}>{`🐙`}</span>{`.`}</undefined></p>
10:48:12 PM:   58 | <p><em parentName="p"><undefined parentName="em">{`-A.Q. `}<span {...{"role":"img","aria-label":""}}>{`👩🏻‍💻`}</span>{` and Canela `}<span {...{"role":"img","aria-label":""}}>{`🐕‍🦺`}</span></undefined></em></p>
10:48:12 PM: > 59 |     </MDXLayout>
10:48:12 PM:      |                 ^
10:48:12 PM:   60 |   )
10:48:12 PM:   61 | };
10:48:12 PM:   62 | MDXContent.isMDXComponent = true
10:48:12 PM: > Build error occurred
10:48:12 PM: Error: > Build failed because of webpack errors
10:48:12 PM:     at nextBuildSpan.traceAsyncFn (/opt/build/repo/node_modules/next/dist/build/index.js:397:19)
10:48:12 PM: npm ERR! code ELIFECYCLE
10:48:12 PM: npm ERR! errno 1
10:48:12 PM: npm ERR! @asyncapi/website@0.1.0 build: `node scripts/index.js && next build`
10:48:13 PM: npm ERR! Exit status 1
10:48:13 PM: npm ERR!
10:48:13 PM: npm ERR! Failed at the @asyncapi/website@0.1.0 build script.
10:48:13 PM: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
10:48:13 PM: npm ERR! A complete log of this run can be found in:
10:48:13 PM: npm ERR!     /opt/buildhome/.npm/_logs/2022-03-22T21_47_48_602Z-debug.log
```
       
Maybe because of the `/` that you use there in the link to your profile. Check locally without it please, or if you need it try `\/` where '\` should escape the `/` character. Not sure though if it will do the trick

Reply:
Nope, I figured it out on my locals. I am reviewing the PR to resolve the build errors.

Reply:
Yeah, it's probably because of the open `<img>` tags above?

Reply:
Yup, I had made a review on it for request changes.

Reply:
@fmvilas @akshatnema The `/` I used to mention my GH profile is actually copy/pasted exactly how I did it in my last blog post with us, and there were no issues there. Ergo, we know that is not the problem :)

Also, <img> tags [do not need a closing tag](https://www.w3.org/wiki/Html/Elements/img). 

Do we have other ideas of what could be causing the issue? :/

Reply:
This is not HTML but JSX. In JSX, everything needs to be closed: https://react-cn.github.io/react/tips/self-closing-tag.html.